### ISSUE TITLE
Missing browser-sync options

### ISSUE THREAD

**Issue by nodeg** [maizzle/framework#231]
- Maizzle Version: 1.1.1
- Node.js Version: 14.4.0

It would be nice to have more flexibility when using browser-sync, e.g. the [ghost-mode](https://browsersync.io/docs/options#option-ghostMode). In my use case it is very annoying that every scroll or click is mirrored if I have another browser tab open and I have to go to the browser-sync UI and disable everything execpt the `CodySync` manually, which is very annoying:
![bs](https://user-images.githubusercontent.com/12104291/86332766-3e2e1d00-bc4b-11ea-94d5-3c49e4b1dd95.png)

The browser-sync initialization is done here
https://github.com/maizzle/framework/blob/38664a5fd8b4b4166e67d7ef0410ea44671d5cdd/src/index.js#L42-L52

and I think this would do the trick:

```js
 bs.init({ 
   server: { 
     baseDir, 
     directory: bsOptions.directory || true 
   }, 
   ui: bsOptions.ui || {port: 3001}, 
   port: bsOptions.port || 3000, 
   notify: bsOptions.notify || false, 
   tunnel: bsOptions.tunnel || false, 
   open: bsOptions.open || false,
   ghostMode: bsOptions.ghostMode || false
 }) 
```

I am not that familiar with browser-sync so I did not create a pull request and want to discuss the possibility of adding more options here.

**Comment by cossssmin** [comment]
You're right, and I think it would be better to just allow passing any Browsersync option from the Maizzle config.

Thanks for bringing this up and don't worry about the PR, I'm on it 👍

**Comment by nodeg** [comment]
Thank you very much!

**Comment by cossssmin** [comment]
I've just published 1.2.0, so you can already update and start using it 👍
